src/main/java/org/sagebionetworks/bridge/validators/AccountIdValidator.java

```diff
@@ -48,6 +49,9 @@ public void validate(Object obj, Errors errors) {
         if (type == ChannelType.EMAIL) {
             if (StringUtils.isBlank(identifier.getEmail())) {
                 errors.rejectValue("email", "is required");
+            } else {
+                // TODO: Check if this is necessary, seems like it is not validated locally, so probably not persisted?
+                validateStringLength(errors, 255, identifier.getEmail(), "email");
```

Comment:
This doesn't seem necessary. Email is validated in the StudyParticipantValidator and this isn't in a path to the DB.

Reply:
Remove it.

Reply:
Done